### PR TITLE
Add region sector lookups to live data

### PR DESCRIPTION
**Pull request by imsickofmaps**
Sectors should be stored in a JS array in the config.

Data for look up is: https://docs.google.com/a/westerncapelabs.com/spreadsheet/ccc?key=0AqDQKdg1X6cPdHp1blVROHFONHdLV195elM4U0FKM1E#gid=0

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@smn ready for review

File: js_sandbox/lib/go-nike-ghr.js [hunk 1]
```diff
@@ -49,6 +49,14 @@ function GoNikeGHR() {
         return p;
     };
 
+    self.validate_sector = function(im, sector){
+        if (im.config.sectors.indexOf(sector.toLowerCase())!=-1){
```
Comment (smn) [inline js_sandbox/lib/go-nike-ghr.js]:
``` javascript
return im.config.sectors.indexOf(sector.toLowerCase()) != -1;
```

would be simpler

File: js_sandbox/test/test-go-nike-ghr.js [hunk 2]
```diff
@@ -33,7 +33,8 @@ describe("When using the USSD line", function() {
...
...ruramba", "rusenge", "bweramana", "byimana", "kabagari", "kinazi", "kinihira", "mbuye", "mwendo", "ntongwe", "ruhango", "bwishyura", "gishari", "gishyita", "gisovu", "gitesi", "kareba", "murambi", "mubuga", "mutuntu", "rubengera", "rugabano", "ruganda", "rwankuba", "twumba", "bwira", "gatumba", "hindiro", "kabaya", "kageyo", "kavumu", "matyazo", "muhanda", "muhororo", "ndaro", "ngororero", "nyange", "sovu", "bigogwe", "jenda", "jomba", "kabatwa", "karago", "kintobo", "mukamira", "muringa", "rambura", "rugera", "rurembo", "shyira", "bushekeri", "bushenge", "cyato", "gihombo", "kagano", "kanjogo", "karambi", "karengera", "kirimbi", "macuba", "mahembe", "nyabitekeri", "rangiro", "ruharambuga", "shangi", "bugeshi", "busasamana", "cyanzarwe", "gisenyi", "kanama", "kanzenze", "mudende", "nyakiliba", "nyamyumba", "nyundo", "rubavu", "rugerero", "bugarama", "butare", "bweyeye", "gikundamvura", "gashonga", "giheke", "gihundwe", "gitambi", "kamembe", "muganza", "mururu", "nkanka", "nkombo", "nkungu", "nyakabuye", "nyakarenzo", "nzahaha", "rwimbogo", "boneza", "gihango", "kigeyo", "kivumu", "manihira", "mukura", "murunda", "musasa", "mushonyi", "mushubati", "nyabirasi", "ruhango", "rusebeya"]
```
Comment (smn) [inline js_sandbox/test/test-go-nike-ghr.js]: can I suggest sticking this in a fixture and loading it like [we do here?](https://github.com/praekelt/mama-ussd/blob/develop/test/test-mama-ussd.js#L33)

**Comment by smn** [comment]
other than minor comments :+1:

**Pull request merged**